properties.py

```diff
@@ -0,0 +1,79 @@
+import json
+import jsonschema
+import gi
+gi.require_version('Gst', '1.0')
+from gi.repository import GObject, Gst, GLib
+
+
+def load_from_json(pipeline, json_file='props.json', schema_file='props_schema.json'):
```

Comment:
maybe add a ret_code so that code using it as a utility will be able to know if the loading succeded or not

Reply:
Done